Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2138,8 +2174,9 @@ void CGContextClearRect(CGContextRef context, CGRect rect) {
     ComPtr<ID2D1CommandList> commandList;
     RETURN_IF_FAILED(deviceContext->CreateCommandList(&commandList));
 
-    deviceContext->BeginDraw();
+    EscapeBeginEndDrawStack();
```

Comment:
#1810

---
In reply to: [97693686](https://github.com/Microsoft/WinObjC/pull/1750#discussion_r97693686) [](ancestors = 97693686)